knora/tmp_xlsx2list.py

```diff
@@ -0,0 +1,84 @@
+import argparse
```

Comment:
Same code is in `onto_commons.py`. Is this temporary file still needed?

Reply:
File removed – no longer needed after pull request